docs/resources/Channel.md

```diff
@@ -893,6 +893,7 @@ Files must be attached using a `multipart/form-data` body as described in [Uploa
 | Field                 | Type                                                                                              | Description                                                                                                                                                                 |
 | --------------------- | ------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | content?\*            | string                                                                                            | Message contents (up to 2000 characters)                                                                                                                                    |
+| nonce?                | integer or string                                                                                 | Can be used to verify a message was sent (up to 25 characters)                                                                                                              |
```

Comment:
the 25 characters only makes sense for the string type, but nonce can also be and int